pkg/controller/controller_broker_test.go

```diff
@@ -1051,3 +1052,106 @@ func TestUpdateServiceBrokerCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestReconcileClusterServicePlanFromClusterServiceBrokerCatalog(t *testing.T) {
+	updatedPlan := func() *v1beta1.ClusterServicePlan {
+		p := getTestClusterServicePlan()
+		p.Spec.Description = "new-description"
+		p.Spec.ExternalName = "new-value"
+		p.Spec.Free = false
+		p.Spec.ExternalMetadata = &runtime.RawExtension{Raw: []byte(`{"field1": "value1"}`)}
+		p.Spec.ServiceInstanceCreateParameterSchema = &runtime.RawExtension{Raw: []byte(`{"field1": "value1"}`)}
+		p.Spec.ServiceInstanceUpdateParameterSchema = &runtime.RawExtension{Raw: []byte(`{"field1": "value1"}`)}
+		p.Spec.ServiceBindingCreateParameterSchema = &runtime.RawExtension{Raw: []byte(`{"field1": "value1"}`)}
+
+		return p
+	}
+
+	// new
+	// update - exists for a different broker
+	// update - ok
+	// update - error updating
```

Comment:
nit: can you put the descriptions for each case above the individual elements in the slice below? not blocking for this PR

Reply:
other people have asked for the opposite - i suggest someone create an issue to figure out how we want to do this instead of hashing it out differently in individual PRs